samples/mycompany.sh

```diff
@@ -27,6 +27,17 @@ An application is a piece of software that
 can be deployed independently in the information system to offer an identified high level service
 '
 
+curl --request PUT \
+  --url $API_BASE_URL/v1/contents/help-what-is-a-contributor/en-GB \
+  --header 'content-type: text/plain' \
+  --data 'Authors
+------
+
+A contributor is someone who commit on code repository.
+
+A maintainer is someone who actively support this application.
+'
+
```

Comment:
I do not agree fully on the definition of a maintainer. We can keep it simple here and put a fake value to close the discussion.

Reply:
What is your definition of a maintainer, just give it to me ... and i update the PR ?